tools/cloudhsm-cleanup/find_ec_keys_by_label.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+
+CLOUDHSM_USER=${CLOUDHSM_USER:?}
+CLOUDHSM_PIN=${CLOUDHSM_PIN:?}
+CLOUDHSM_USER_TYPE=${CLOUDHSM_USER_TYPE:-"CU"}
+
+# read key labels from stdin
+# search for EC public and private keys matching the label
```

Comment:
> Why are we restricting search to EC keys?

Yes! Content signature keys are EC so we limit our search that type to avoid accidentally deleting another key type (like an RSA key for MAR updates).

> iirc, key labels must be unique

Key labels are not unique to a single key handle. We have the handle for the private key from the DB `end_entities` row, but we have to search by label to find the corresponding handle for the public key.

Reply:
> Key labels are not unique to a single key handle. 

You're right, they don't even need to be unique to a key pair! (I even [added that note](https://docs.google.com/document/d/1tVYApGVvSSyMnBrWBryev6VA_1kHm4l1XRHm9rGSZ24/edit#heading=h.b67g9mnrpaag)!)